verdictd/Makefile

```diff
@@ -0,0 +1,46 @@
+
+PROJDIR := $(shell readlink -f ..)
+TOP_DIR := .
+#include $(TOP_DIR)/buildenv.mk
+
+######## APP Settings ########
+
+App_Rust_Flags := --release
+App_SRC_Files := $(shell find app/ -type f -name '*.rs') $(shell find app/ -type f -name 'Cargo.toml')
+
+App_Rust_Path := ./app/target/release
+Opa_Lib_Path := ./app/src/policyEngine/opa
+App_Name := verdictd
+Opa_Name := libopa.so
+
+.PHONY: all
+all: $(App_Name)
+
+######## App Objects ########
+$(App_Name): $(App_SRC_Files)
+	@cd app && RUSTFLAGS="-C link-args=-Wl,-rpath=/opt/enclave-tls/lib:/usr/local/lib:$(PROJDIR)/verdictd/app/src/policyEngine/opa,--enable-new-dtags" cargo build $(App_Rust_Flags)
+	@echo "Cargo  =>  $@"
+	mkdir -p bin lib
+	cp $(App_Rust_Path)/$(App_Name) ./bin
+	cp $(Opa_Lib_Path)/$(Opa_Name) ./lib
+
+.PHONY: install uninstall clean
+
+PREFIX := /usr/local
+BINDIR := $(PREFIX)/bin
+LIBDIR := $(PREFIX)/lib
+
+install: $(App_Name)
+	@install -D -m0755 ./bin/$(App_Name) "$(BINDIR)"
+	@install -D -m0755 ./lib/$(Opa_Name) "$(LIBDIR)"
+
+uninstall:
+	@rm -f $(BINDIR)/$(App_Name)
+	@rm -f $(LIBDIR)/$(Opa_Name)
+
+#package:
+#	$(MAKE) -C dist package
+
+clean:
+	@rm -rf ./bin/ ./lib/ ./app/src/policyEngine/opa/libopa.*
```

Comment:
You can use $(Opa_Lib_Path)/$(Opa_Name) to replace `./app/src/policyEngine/opa/libopa.*`

Reply:
I will change to $(Opa_Lib_Path)/libopa.* because there is also a libopa.h file need to be deleted.